public/javascripts/shoppinglist.js

```diff
@@ -0,0 +1,179 @@
+"use strict";
+/*global React*/
+
+var FormPage = React.createClass({
+  getInitialState: function() {
+    return({ShopList: []});
+  },
+  componentDidMount: function() {
+
+    //BTW: currently loads most recent list on refresh
+
+    $.getJSON('/shoppinglist/getlist', function(data) {
+      this.setState({ShopList: data});
+    }.bind(this));
+  },
+  render: function() {
+    var showList = [];
```

Comment:
More descriptive name (this is a builder for the DOM Node representing the list right?)